libs/atqr/domain/src/lib/types/notification.types.ts

```diff
@@ -0,0 +1,32 @@
+import { ITemplateDataTypes } from '../services/notification/templateDataType.interface';
+
+export type TemplateDataTypes = {
+  [K in keyof ITemplateDataTypes]: ITemplateDataTypes[K];
+};
+
+export enum MessageTemplateId {
+  ChallengeCreated = 'ChallengeCreated',
+  ChallengeCanceled = 'ChallengeCanceled',
+  ChallengeOngoing = 'ChallengeOngoing',
+  ChallengeCompleted = 'ChallengeCompleted',
+  ChallengeFailed = 'ChallengeFailed',
+  YourOfficiationHaveBeenRequested = 'YourOfficiationHaveBeenRequested',
+  YourOfficiationRequestHaveBeenAccepted = 'YourOfficiationRequestHaveBeenAccepted',
+  YourOfficiationRequestHaveBeenRejected = 'YourOfficiationRequestHaveBeenRejected',
+  YourOfficiationRequestHaveBeenIgnored = 'YourOfficiationRequestHaveBeenIgnored',
+  YouHaveBeenChallenged = 'YouHaveBeenChallenged',
+  YourContenderAcceptedTheChallenge = 'YourContenderAcceptedTheChallenge',
```

Comment:
The disl definition for this field is duplicated. Let's try to keep both Disl and TS in sync ;)